.github/actions/action-lacework-scan/action.yml

```diff
@@ -0,0 +1,114 @@
+name: 'Run Lacework image scan'
+description: 'Scan the provided image with Lacework scan tool and return a JSON with the results.'
+
+inputs:
+  image:
+    description: 'Full URL of the image to scan.'
+    required: true
+  docker-username:
+    description: 'Docker username.'
+    required: true
+  docker-password:
+    description: 'Docker password.'
+    required: true
+  lw-account-name:
+    description: 'Lacework account name.'
+    required: true
+  lw-access-token:
+    description: 'Lacework access token.'
+    required: true
+  severity-filter:
+    description: 'Specify severities of CVEs to be included in the result, in CSV format, e.g. "Critical,High,Medium,Low,Info". (Default: "Critical,High")'
+    default: 'Critical,High'
+  save-results-in-lacework:
+    description: 'Save results to Lacework. (Default: false)'
+    default: 'false'
+
+outputs:
+  result-json:
+    description: 'Lacework scan result in JSON format.'
+    value: ${{ steps.process-result.outputs.result-json }}
+
+runs:
+  using: "composite"
+  steps:
+  - name: Prepare input
+    id: prepare-input
+    shell: bash
+    run: |
+      echo "image-name=$(echo "${{ inputs.image }}" | cut -d: -f1)" >> $GITHUB_OUTPUT
+      echo "image-tag=$(echo "${{ inputs.image }}" | cut -d: -f2)" >> $GITHUB_OUTPUT
+      echo "safe-image-id=$(echo "${{ inputs.image }}" | sed 's/[^a-zA-Z0-9.-]/-/g')" >> $GITHUB_OUTPUT
+
+  - name: Docker login
+    uses: docker/login-action@v2
+    with:
+      username: ${{ inputs.docker-username }}
+      password: ${{ inputs.docker-password }}
+
+  - name: Docker pull
+    shell: bash
+    run: docker pull ${{ inputs.image }}
+
+  - name: Scan
+    uses: lacework/lw-scanner-action@v1.1.1
+    with:
+      LW_ACCOUNT_NAME: ${{ inputs.lw-account-name }}
+      LW_ACCESS_TOKEN: ${{ inputs.lw-access-token }}
+      IMAGE_NAME: ${{ steps.prepare-input.outputs.image-name }}
+      IMAGE_TAG: ${{ steps.prepare-input.outputs.image-tag }}
+      SCAN_LIBRARY_PACKAGES: true
+      SAVE_RESULTS_IN_LACEWORK: ${{ inputs.save-results-in-lacework }}
+      LW_SCANNER_ENABLE_DEBUGGING: ${{ env.ACTIONS_STEP_DEBUG }}
+
+
+  - name: Process result
+    id: process-result
+    shell: bash
+    run: |
+      fullJson=$(find ./evaluations -name 'evaluation_*.json' 2>/dev/null)
```

Comment:
It should always find only a single file, as the job runs in a clean work directory and the scan generates only a single file. If it will return more than 1 files then further steps will fail and we will at least know about it.

Reply:
IMHO it's always better to act as if it is a list. However, not a big issue in this case..